enterprise/backend/test/metabase_enterprise/serialization/v2/e2e/yaml_test.clj

```diff
@@ -108,9 +108,7 @@
             (doseq [{:keys [name] :as coll} (get entities "Database")
                     :let [filename (#'u.yaml/leaf-file-name name)]]
               (is (= (-> coll
-                         (dissoc :serdes/meta)
-                         (update :created_at u.date/format)
-                         (update :updated_at u.date/format))
+                         (dissoc :serdes/meta :created_at :updated_at))
```

Comment:
Is this worth abstracting out into a function somewhere so we don't have to repeat it dozens of times and then update it dozens of times as well? You could call it `dissoc-irrelevant-keys` or something like that.

Reply:
Done.